src/mcp_server_starrocks/server.py

```diff
@@ -540,12 +540,12 @@ def query_and_plotly_chart(query: Annotated[str, Field(description="SQL query to
         # with open("chart.jpg", "wb") as f:
         #     f.write(img_bytes)
         # base64 encode the image bytes
-        img_base64_bytes = base64.b64encode(img_bytes)
+        # img_base64_bytes = base64.b64encode(img_bytes)
         # Decode bytes to utf-8 string for easier handling (e.g., JSON serialization)
-        img_base64_string = img_base64_bytes.decode('utf-8')
+        # img_base64_string = img_base64_bytes.decode('utf-8')
         return [
             f'dataframe data:\n{df}\nChart generated but for UI only',
-            Image(data=img_base64_string, format="jpg")
+            Image(data=img_bytes, format="jpg")
```

Comment:
looking at fastmcp code:

```

    def _get_mime_type(self) -> str:
        """Get MIME type from format or guess from file extension."""
        if self._format:
            return f"image/{self._format.lower()}"

        if self.path:
            suffix = self.path.suffix.lower()
            return {
                ".png": "image/png",
                ".jpg": "image/jpeg",
                ".jpeg": "image/jpeg",
                ".gif": "image/gif",
                ".webp": "image/webp",
            }.get(suffix, "application/octet-stream")
        return "image/png"  # default for raw binary data
```

there is also a bug for the format arg, should be jpeg instead of jpg

Image(data=img_bytes, format="jpeg")

please fix this together?